Archives/VolFile.cpp

```diff
@@ -378,49 +387,37 @@ namespace Archives
 	}
 
 	// Writes a section tag to the open output file.
-	void VolFile::WriteTag(StreamWriter& volWriter, uint32_t length, const char *tagText)
+	void VolFile::WriteTag(StreamWriter& volWriter, uint32_t length, const std::array<char, 4>& tagName)
 	{
-		int buffer[2];
-
-		buffer[0] = *(int*)tagText;
-		// Set padding bit to indicate 4 byte padding
-		buffer[1] = length | 0x80000000;
+		SectionHeader sectionHeader(tagName, length);
```

Comment:
I set the constructor so if the padding value is not added then it will automatically be set to 1. 

What do you think of this format, where an argument has a default value? Maybe it isn't appropriate here?

Reply:
You're right. I didn't see that until after I submitted the review. I was focused on the implementation of the constructor, where the default value isn't listed.

The code is fine as is.